app/commands/v2/put_link_set.rb

```diff
@@ -11,6 +11,8 @@ def call
 
           links_hash = link_params.fetch(:links)
 
+          link_set.links.delete_all if links_hash.empty?
```

Comment:
I don't think this is the right behaviour. The links endpoint is meant to have patch semantics, only updating the link types that are actually sent. In particular we need to support the world when only some of the links are managed by Whitehall, and the rest are managed by content-tagger.